.envrc

```
@@ -216,11 +223,7 @@ PATH_add node_modules/.bin
 
 # These are commands that can take a significant amount of time
 if [ ${#commands_to_run[@]} -ne 0 ]; then
-    echo -e "\n${red}Run the following commands to bring your environment up-to-date:"
-    for cmd in "${commands_to_run[@]}"; do
-        warn "    ${red}$cmd"
-    done
-    echo ""
+    commands_info
```

I like having a verb in here to better show what it's doing e.g.

```suggestion
    show_commands_info
```